app/src/main/java/com/example/campusguide/BottomNavigation.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.campusguide
+
+import android.content.Intent
+import com.example.campusguide.directions.ChooseDestinationOptions
+import com.example.campusguide.directions.ChooseOriginOptions
+import com.example.campusguide.location.FusedLocationProvider
+import com.example.campusguide.map.GoogleMapAdapter
+import com.example.campusguide.utils.permissions.Permissions
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.maps.model.LatLng
+
+class BottomNavigation constructor (
+    private val activity: MapsActivity,
+    private val map: GoogleMapAdapter,
+    private val permissions: Permissions,
+    private val locationProvider: FusedLocationProvider) {
+
+    private lateinit var bottomNavigationView : BottomNavigationView
+
+    fun setupBottomNavigation() {
+        bottomNavigationView = activity.findViewById(R.id.bottom_navigation)
+        bottomNavigationView.selectedItemId = R.id.campus_map
+        bottomNavigationView.setOnNavigationItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.directions -> {
+                    startNavigation()
+                }
+                R.id.campus_map -> {
+                    centerCameraSGW()
+                }
+                R.id.poi -> {
+                    // points of interest view
```

Comment:
Should we put a TODO here? Or is that not appropriate since POI view is a feature, not technical debt?

Reply:
POI view is a new view I'm thinking of designing to fulfill #24, I could add a TODO if it's appropriate in that case

Reply:
I'll add the TODO just as a reminder!